### DISCUSSION TITLE
What happens if we allocation 3 stations per stratum instead of 2?

### DISCUSSION THREAD

**Discussion by MargaretSiple-NOAA** [afsc-gap-products/StationAllocationAIGOA#22] [Ideas]
Ned wants to know if setting the min numbers of stations per stratum to 3 instead of 2 will give us "new stations" (additional, beyond the original/required allocation) that are allocated more organically than the way we did it in 2022. I am putting this here because I don't know what will happen and it seems like something we can experiment with for 2024.

In his words:

> The simplest way we can improve our variance estimates in difficult strata is to require 3 instead of 2 stations. I expect that in the AI if we bump from 2 to 3 as the minimum, we'll start to get more "new stations" generated straight out of the allocation rather than have to force feed them after the fact. If we move in this direction, we'll need to be cognizant of any new stations coming out of the formal allocation and include them in the 10-12 total new stations assigned.

**Comment by zoyafuso-NOAA** [comment]
By new AI stations, we mean stations that have NA in the trawlable field, right? I think in the [sql call](https://github.com/afsc-gap-products/StationAllocationAIGOA/blob/c05e353611cc78fa0f2c72fc3cfe7e590d0f3b28/R/get.ai.stations.R#L57) that queries available stations for the AI allocation, stations with a NA/NULL trawlable status are filtered out. So increasing the minimum number of  stations argument won't necessarily incorporate new stations. 

As a side, I am in favor of increasing the min number of stations per stratum but it's more to create a buffer against having a one-station-stratum situation.

**Reply by MargaretSiple-NOAA** [reply]
Thanks Zack. I will leave it to @Ned-Laman-NOAA to say whether he is thinking of stations that have NA in the trawlable field; I'm not 100% sure. 

You're right, I think increasing min stations per stratum is a good idea to make a buffer; currently we have a min of 2 assigned but that means that if for any reason someone misses one (which hasn't happened for a long time, but I think it's still possible, especially in the AI!), we have one station in that stratum.

**Comment by Ned-Laman-NOAA** [comment]
Let's set a timer on this one to go off in 2024 when we're discussing Aleutian station allocation again. I do think we should bump up the minimum number of stations per stratum particularly in this region.
